packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx

```diff
@@ -234,7 +234,8 @@ describe('<BottomSheet />', () => {
     mockConsoleError.mockRestore();
   });
 
-  it('should compose with Dropdown single select', async () => {
+  // TODO: Dropdown hasn't been rebranded yet thus skipping this test until then
```

Comment:
Easier to global search later on
```suggestion
  // TODO: Rebranding - Dropdown hasn't been rebranded yet thus skipping this test until then
```

Reply:
done.